features/events/traps/pom.xml

```diff
@@ -76,11 +76,6 @@
     </plugins>
   </build>
   <dependencies>
-   <dependency>
-        <groupId>javax.ws.rs</groupId>
-        <artifactId>javax.ws.rs-api</artifactId>
-        <version>2.0.1</version>
```

Comment:
@RangerRick : This has been deleted but when we  tried to trace back for this change we were  unable to find in any previous commit.This looks bit strange.